src/PresentationalComponents/Notification/Notification.js

```diff
@@ -19,7 +19,7 @@ import {
   TextContent,
   TextVariants,
 } from '@patternfly/react-core';
-import { componentMapper } from '@data-driven-forms/pf4-component-mapper/component-mapper/component-mapper';
+import componentMapper from '@data-driven-forms/pf4-component-mapper/component-mapper/component-mapper';
```

Comment:
Do not use second level imports. `@data-driven-forms/pf4-component-mapper/component-mapper` is enough. Deep imports could cause esm/cjs import mismatch